### ISSUE TITLE
Language localization

### ISSUE THREAD

**Issue by nulijiazaizhong** [CoffeSiberian/truck-tools#5]
I see that the language options are currently only in English and cannot be changed, so I hope to be able to localize as soon as possible, which is very important for us, and if you need localization, I can help with the localization of Chinese Simplified

**Comment by CoffeSiberian** [comment]
At the moment I have not started to work with the translations but in the future I will have the templates ready to be translated into more languages. At the moment I haven't started with this because of time issues

**Comment by nulijiazaizhong** [comment]
Okay, thank you very much for your answer, but I'd still like you to tell me a rough timeframe

**Comment by CoffeSiberian** [comment]
I would like to have the templates and frontend context functional by the end of November. It's not something that's going to take me a long time, but I have other priorities (other projects at the University) at the moment. I will try to do my best to have it before

**Comment by xingnengmao666** [comment]
The interface language seems to be hard-coded into the software now, so hopefully a template for i18n will be released soon!

**Comment by CoffeSiberian** [comment]
The templates will be available soon. I haven't had the time because of my thesis 👀☕

**Comment by xingnengmao666** [comment]
Understanding. My father is also struggling with this :(

**Comment by MRUIAW** [comment]
> I see that the language options are currently only in English and cannot be changed, so I hope to be able to localize as soon as possible, which is very important for us, and if you need localization, I can help with the localization of Chinese Simplified我看到语言选项目前只有英文，无法更改，所以希望能够尽快本地化，这对我们来说非常重要，如果您需要本地化，我可以帮助本地化中文简化版

oh u are also here :D

**Comment by CoffeSiberian** [comment]
That's right, I always try to respond to every comment I receive on any platform. As I said, let me finish my thesis and I can start again with new tasks!

**Comment by CoffeSiberian** [comment]
Now have the possibility to start suggesting translations. You can read the following instructions https://github.com/CoffeSiberian/truck-tools?tab=readme-ov-file#contribute-translations
